# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../loop-aes/kernel-2.6.19.1.diff
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- kernel-2.6.18.diff.orig	2006-12-22 14:43:49.000000000 +0100
+++ kernel-2.6.18.diff	2006-12-22 15:49:00.000000000 +0100
@@ -21,15 +21,16 @@
  	  Note that this loop device has nothing to do with the loopback
  	  device used for network connections from the machine to itself.
  
-@@ -301,19 +293,32 @@
+@@ -302,20 +294,34 @@
  
  	  Most users will answer N here.
  
 -config BLK_DEV_CRYPTOLOOP
 -	tristate "Cryptoloop Support"
--	select CRYPTO
 +config BLK_DEV_LOOP_AES
 +	bool "AES encrypted loop device support"
+ 	select CRYPTO
+ 	select CRYPTO_CBC
  	depends on BLK_DEV_LOOP
  	---help---
 -	  Say Y here if you want to be able to use the ciphers that are 
